packages/components/src/components/telekom/telekom-header/telekom-header.tsx

```diff
@@ -47,7 +47,7 @@ export class TelekomHeader {
 
   @Listen('scroll', { target: 'document' })
   onScroll() {
-    this.scrolled = window.pageYOffset > 2;
+    this.scrolled = window.pageYOffset > 48;
```

Comment:
I would prefer this 48 not be hard-coded, but calculated… if not possible (or too complex), at least add a comment explaining where the number is coming from 🤓

Reply:
i thought about calculating it here but it seemed rather overkill for this. Only issue i see with this now, is that on medium viewport the header is not 48px but a little bit smaller - IMO not a problem, it would just minimize when scrolling 10px more than the header height at that viewport size. (or was it requested by UX/UI that the header minimizes ASAP? )